File: jdroid-android-google-inappbilling/src/debug/java/com/jdroid/android/google/inappbilling/InAppBillingDebugAppLifecycleCallback.java
package com.jdroid.android.google.inappbilling;

import android.content.Context;

import com.jdroid.android.debug.DebugSettingsHelper;
import com.jdroid.android.lifecycle.ApplicationLifecycleCallback;

public class InAppBillingDebugAppLifecycleCallback extends ApplicationLifecycleCallback {

	@Override
	public void onProviderInit(Context context) {
		DebugSettingsHelper.addPreferencesAppender(new InAppBillingDebugPrefsAppender());
	}

}

File: jdroid-android-google-inappbilling/src/debug/java/com/jdroid/android/google/inappbilling/InAppBillingDebugAppLifecycleCallback.kt
package com.jdroid.android.google.inappbilling

import android.content.Context

import com.jdroid.android.debug.DebugSettingsHelper
import com.jdroid.android.lifecycle.ApplicationLifecycleCallback

class InAppBillingDebugAppLifecycleCallback : ApplicationLifecycleCallback() {

    override fun onProviderInit(context: Context) {
        DebugSettingsHelper.addPreferencesAppender(InAppBillingDebugPrefsAppender())
    }
}
